### PR TITLE
Update the install instructions

### PR DESCRIPTION
**Pull request by richvdh**
To include some of the newer components.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -108,23 +108,50 @@ The following contains scripts which will run all the required processes in orde
...
+                                   |    |     +--------------------------+                 ||
+Matrix      +------------------+   |    |   :7773    |                                     ||
+Clients --->| client-api-proxy |---+    +<-----------+                                     ||
+            +------------------+   |    |                                                  ||
+          :8008                    | CS API   +----------------------------+               ||
+                                   +--------->| dendrite-client-api-server |==++           ||
+                                        |     +----------------------------+  ||           ||
+                                        |   :7771    |                        ||           ||
+                                        |            |            client_data ||           ||
+                                        +<-----------+         ++=============++           ||
```
Comment (NegativeMjark) [inline INSTALL.md]: I think the client_data stream goes to the sync-api-server not the room-server

File: INSTALL.md [hunk 2]
```diff
@@ -108,23 +108,50 @@ The following contains scripts which will run all the required processes in orde
...
+                                        |     +-----------------------------------+              Matrix
+                                        |     | dendrite-federation-sender-server |------------> Servers
+                                        |     +-----------------------------------+
+                                        |   :7776
+                                        |
+                                        +<-----------+
+                                                     |
+Matrix      +----------------------+  SS API  +--------------------------------+
+Servers --->| federation-api-proxy |--------->| dendrite-federation-api-server |
+            +----------------------+          +--------------------------------+
```
Comment (NegativeMjark) [inline INSTALL.md]: Technically the federation-api-proxy talks to the media-api-server as well...

File: INSTALL.md (hunk 1 above)
Comment (richvdh) [reply INSTALL.md]: So it does. fixed in 46e0a66

File: INSTALL.md (hunk 2 above)
Comment (richvdh) [reply INSTALL.md]: Are you sure? how?

Comment (NegativeMjark) [reply INSTALL.md]: Oh, I guess no one implemented that yet. It SHOULD talk to the media-api-server as well :)

Comment (NegativeMjark) [reply INSTALL.md]: I'll fix that in a separate PR I guess.

**Pull request merged**